roles/edpm_libvirt/templates/firewall.yaml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
Is this an example of the template rendering? If so, it should be moved into docs.

Reply:
no it is not.
it is the template that is used to render the firewall config
not an example